pallets/invulnerables/src/benchmarking.rs

```diff
@@ -112,6 +112,7 @@ pub(crate) fn currency_issue<T: Config + frame_system::Config>(
     <<T as crate::Config>::Currency as Balanced<T::AccountId>>::issue(amount)
 }
 
+#[allow(clippy::multiple_bound_locations)]
```

Comment:
For this one, I guess it's part of the macro and we cannot do much

Reply:
Yeah, it's the one downside of upgrading Rust before upgrading polkadot-sdk, we see clippy warnings from code generated by macros...